### ISSUE TITLE
hide events that are in the future from rss feeds

### ISSUE THREAD

**Issue by derpeter** [voc/voctoweb#450]
Currently it is possible to create events with a date that is in the future. This results in RSS feeds showing this event always on top until we reach that date and newer events are submitted.

Until time travel is widely available we should check this in the API and reject the creation of the event.

**Comment by saerdnaer** [comment]
we actually had to remove this check, because with relive events in future are possible – but we have to hide such event without recordings from the rss feeds.

**Comment by derpeter** [comment]
hiding events without recording is not solving the issue. Events with a recording can still be in the future and would therefore be the newest in the RSS feed until time passed past the date.

We would need to hide all events with a timestamp in the future from RSS to solve the issue.
Not sure if we need relive events in the RSS so this maybe a solution. Or we need a threshold like "hide everthing that is not happening in the next 2 hours"

**Comment by derpeter** [comment]
I changed the title again but in fact i'm unhappy with changing the title without waiting for the OP to agree that issue is fully understood and a solution is agreed.

**Comment by saerdnaer** [comment]
Let me rephrase my sentence from before: The actual filter used, e.g. in on the start page, is to display only events with an non-empty publication date.

**Comment by derpeter** [comment]
I don't get where this is related to issue. I get that you need events with future dates for relive but in the case that was the reason for opening the issue we have a publication date and recordings but the date ist just wrong.
